### ISSUE TITLE
Touch affordances

### ISSUE THREAD

**Issue by stuaxo** [steffest/DPaint-js#12]
Had a quick play on an Android phone and wonder how touch affordances could work.

There's a question of when to enable them, I'm not sure if there are APIs that help find the size of the screen, which might help.

In the case of sliders for instance l, having a larger UI for it that appears when you touch the screen could be one place to start.

**Comment by steffest** [comment]
Yeah... for small screen devices we need a completely different UI.
I implemented the "bare minimum" for now and kind of ignored all the rest until the feature set is complete.

**Comment by stuaxo** [comment]
I was surprised at how usable some of it is TBH.   It's definitely possible to press the larger icons, though maybe two columns of them isn't ideal.

You're right that piecemeal isn't ideal... this is a bit of fluffy ticket anyway :)

**Comment by stuaxo** [comment]
For reference here's a phone screenshot (smaller, in reality):
![Screenshot_20230603-130057](https://github.com/steffest/DPaint-js/assets/179677/811c4c90-3e23-4224-8b40-67ee27be5f64)

**Comment by fccfc5d245c** [comment]
I can use it on a phone with the current UI size, what I find fustrating (and I don't know if it's a bug, that's why I'm sharing) is that I can't touch-drag to move the slider or color selector on the Color side-pane . I mean: I have to touch the exact spot where the color I want is, if I touch and drag, I drag the whole panels group. 

This could be solved by adding HSB sliders below the pane, as all the other sliders I've tried work fine.

**Comment by stuaxo** [comment]
In portrait orientation it might make sense for the icons to be along the bottom (especially on a phone as that's nearer your thumbs).

Touch is definitely a bit more than button size and placement, but this might help as an incremental improvement.
